WinUIXamlCompiler/WinUIXamlCompilerExtensions.cs

```diff
@@ -19,7 +21,31 @@ void InsertAfter<T>(params IXamlAstTransformer[] t)
 
             void InsertBefore<T>(params IXamlAstTransformer[] t)
                 => compiler.Transformers.InsertRange(compiler.Transformers.FindIndex(x => x is T), t);
+
+            InsertAfter<ResolveContentPropertyTransformer>(
+                new XamlDirectPropertyResolver(),
```

Comment:
how do you know where to insert certain resolvers at?

Reply:
Mostly experience. The idea here was "I want to do something after I've resolved all properties but before I've figured out how I can assign the values" so I put it after the last phase that resolves properties. I could have also put it before the transform that resolves assignments (which IIRC would be the same place in the pipeline).

Reply:
There's a few phases that are pretty common ones to put things before/after. Take a look at Avalonia's compiler for some examples.